docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,128 @@ platforms: ['android', 'ios', 'tvos']
 import APISection from '~/components/plugins/APISection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
-import { ConfigReactNative, ConfigPluginExample } from '~/components/plugins/ConfigSection';
+import { Collapsible } from '~/ui/components/Collapsible';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { GithubIcon } from '@expo/styleguide-icons';
 
-The `expo-updates` library allows you to programmatically control and respond to new updates made available to your app.
+## Installation & Configuration
 
-## Installation
+`expo-updates` is a library that enables your app to manage remote updates to your application code. It communicates with the configured remote update service to get information about available updates.
+
+[EAS Update](/eas-update/introduction) is a hosted service that manages and serves updates, and includes tools to simplify installation and configuration of this library. To get started with EAS Update, follow the [Getting Started Guide](/eas-update/getting-started/).
+
+If you're installing this library in a [bare React Native app](/bare/overview/), follow these [additional installation instructions](/bare/installing-updates/).
+
+It is also possible to configure this library manually in cases where native code is configured manually or use of a different remote update service is required.
+
+<Collapsible summary="Manual installation & configuration">
 
 <APIInstallSection hideBareInstructions={true} />
 
-If you're installing this in a [bare React Native app](/bare/overview/), you should also follow these [additional installation instructions](/bare/installing-updates/).
+If you're installing this library in a [bare React Native app](/bare/overview/) or a generic app with manually-configured native code, follow these [installation instructions](/bare/installing-updates/).
 
-<ConfigReactNative>
+If using [app config](/workflow/configuration/) for configuration, this library can be configured by setting at least the following app config keys:
```

Comment:
```suggestion
If using [app config](/workflow/configuration/) for configuration, this library can be configured by setting at least the following app config properties:
```

We prefer to use properties over keys when it comes to app config properties:
![CleanShot 2024-06-03 at 21 36 15@2x](https://github.com/expo/expo/assets/10234615/59cd073b-f2a0-40db-ae7a-87b4dff71dc7)